DataFormats/BTauReco/interface/PixelClusterTagInfo.h

```diff
@@ -0,0 +1,87 @@
+#ifndef DataFormats_BTauReco_PixelClusterTagInfo_h
+#define DataFormats_BTauReco_PixelClusterTagInfo_h
+
+#include "DataFormats/BTauReco/interface/BaseTagInfo.h"
+
+namespace reco {
+
+  struct PixelClusterProperties {
+    float x = 0;
+    float y = 0;
+    float z = 0;
+    int charge = 0;
+    int layer = 0;
+  };
+
+  struct PixelClusterData {
+    char L1_R004 = 0;
+    char L2_R004 = 0;
+    char L3_R004 = 0;
+    char L4_R004 = 0;
+
+    char L1_R006 = 0;
+    char L2_R006 = 0;
+    char L3_R006 = 0;
+    char L4_R006 = 0;
+
+    char L1_R008 = 0;
+    char L2_R008 = 0;
+    char L3_R008 = 0;
+    char L4_R008 = 0;
```

Comment:
at least the layer items better be in an array

Reply:
Ok, i implemented both changes in the latest commit.